test_sched_slack_bot/views/test_schedule_dialog.py

```diff
@@ -47,7 +47,8 @@ def test_get_create_schedule_dialog() -> None:
     assert create_block.blocks[0].text is not None
     assert CREATE_MODAL_TYPE in create_block.blocks[0].text.text
 
-    assert create_block.external_id == CREATE_NEW_SCHEDULE_VIEW_ID
+    if create_block.external_id is not None:
+        assert create_block.external_id.startswith(CREATE_NEW_SCHEDULE_VIEW_ID)
```

Comment:
```suggestion
    assert create_block.external_id is not None
    assert create_block.external_id.startswith(CREATE_NEW_SCHEDULE_VIEW_ID)
```

that way the test would fail if the external_id is None which is probably what we want